### ISSUE TITLE
Compiler Warning when compiling argtable3.c

### ISSUE THREAD

**Issue by pwpiwi** [Proxmark/proxmark3#744]
With gcc 6.3.0 I get the following warning
```
cliparser/argtable3.c: In function 'warnx':
cliparser/argtable3.c:392:0: warning: ignoring #pragma warning  [-Wunknown-pragmas]
 #pragma warning(suppress: 6053)
```
Any idea how to fix?

**Comment by merlokk** [comment]
maybe just delete pragma?
it for supress next line warning

**Comment by merlokk** [comment]
it looks it will be better than delete

**Comment by pwpiwi** [comment]
Will fix next year.

**Comment by pwpiwi** [comment]
Fixed with PR #746
